### ISSUE TITLE
dota 2 stuck on loading screen when launching 

### ISSUE THREAD

**Issue by 13adada** [ValveSoftware/Dota2-Gameplay#2680]
### Description

I've been playing Dota on a MacBook Air 2013 for years now. I've never had any trouble launching or playing the game except for bad graphics. after the 2022 battle pass update, my game wouldn't launch anymore. I've checked the system requirements and they are fine. I've uninstalled and reinstalled the game twice and it still wouldn't work. I erased all the storage on the Mac and installed the game again and it still wouldn't work. I updated the Mac to the newest software update and it wouldn't work. I also tried launching the game in almost every custom launch option there is and nothing worked. please help me asap I need to grind mmr

**Comment by dobstotev** [comment]
yeah i have exactly the same problem

**Comment by 13adada** [comment]
> yeah i have exactly the same problem

pleaaaaase let me know when you find a solution asap

**Comment by dobstotev** [comment]
I think we have to wait for an update from Valve ... don't think its a tech solution on our sides.

**Comment by dobstotev** [comment]
I don't know, an update where Valve & Dota 2 team resolve the issue and update the game so that mac users can play it. There hasn't been an announcement that Mac will no longer be supported or any of that sorts so it might take a couple of days to get a fix since the majority of their players are on windows, mac users are always left behind haha. I'd calm down and wait.

**Comment by RoeelCohen** [comment]
It's for all mac users

**Comment by 13adada** [comment]
> I don't know, an update where Valve & Dota 2 team resolve the issue and update the game so that mac users can play it. There hasn't been an announcement that Mac will no longer be supported or any of that sorts so it might take a couple of days to get a fix since the majority of their players are on windows, mac users are always left behind haha. I'd calm down and wait.



> It's for all mac users

unfortunately I though I was the only one and I erased and reset the Mac and erased all my files on it to clear storage to download the new software update thinking it would solve the issue, aside from uninstalled and reinstalling the game twice, anyways is there somewhere to write a steam report or something maybe valve isn't aware of this issue?

**Comment by MariusRossouw** [comment]
same issue

**Comment by dobstotev** [comment]
> > I don't know, an update where Valve & Dota 2 team resolve the issue and update the game so that mac users can play it. There hasn't been an announcement that Mac will no longer be supported or any of that sorts so it might take a couple of days to get a fix since the majority of their players are on windows, mac users are always left behind haha. I'd calm down and wait.
> 
> > It's for all mac users
> 
> unfortunately I though I was the only one and I erased and reset the Mac and erased all my files on it to clear storage to download the new software update thinking it would solve the issue, aside from uninstalled and reinstalling the game twice, anyways is there somewhere to write a steam report or something maybe valve isn't aware of this issue?

Yeah, you can submit a report via the Steam Mac App by opening a ticket. You can google where exactly to do it. I already submitted one but haven't received a response.

**Comment by 13adada** [comment]
I really hope its fixed soon its been around 15 hours since the game hasn't worked for me I can't imagine what it would be like if windows users had the same issue I think they'd instantly handle it

**Comment by matias2990** [comment]
same with windows here

**Comment by dobstotev** [comment]
> same with windows here

Wait you have the same problem on windows? :O

**Comment by matias2990** [comment]
here just fixed without do it anything

**Comment by knigadima** [comment]
Hey

The same problem, Dota 2 not opening after battle pas 2022 update. Reinstalled Dota but it did not work :(

OS: macOS Catalina 10.15.7
Device: MacBook Air (13-inch, 2017)
Memory: 8 GB 1600 MHz DDR3
Graphics: Intel HD Graphics 6000 1536 MB
Processor: 1,8 GHz double core Intel Core i5

@ValveSoftware please help guys

**Comment by 13adada** [comment]
alright I just saw on dota 2 discussions on steam that Mac users are all familiar with this issue after the update so hopefully this issue will be resolved really soon when valve is aware of it.

**Comment by cryptolove123** [comment]
please fix this.

**Comment by atif-rehman-ideofuzion** [comment]
Valve is too busy fixing this I guess that is why there is no response here lol.

**Comment by 13adada** [comment]
any update? its almost been 24 hours this is getting ridiculous

**Comment by 13adada** [comment]
its up boys

**Comment by 13adada** [comment]
-vulkan -safe -high -sw -map dota -noprewarm

**Comment by 13adada** [comment]
never mind it worked once then stopped

**Comment by fahim0173** [comment]
> -vulkan -safe -high -sw -map dota -noprewarm

This worrks for me but I have to restart my macbook before I launch the game lol.  This is what I did - 

- Validate game integrity
- Set the launch options -vulkan -safe -high -sw -map dota -noprewarm
- Restart Macbook
- Start steam and launch game

Worked for me twice lol
